tools/topology/topology2/cavs/cavs-nocodec.conf

```diff
@@ -39,6 +39,7 @@ Define {
 	DMIC0_HOST_PIPELINE_SINK	'copier.host.13.1'
 	DMIC0_DAI_PIPELINE_SRC		'copier.DMIC.14.1'
 	DMIC0_NAME			'NoCodec-6'
+	DMIC1_NAME			'NoCodec-7'
```

Comment:
I find the notation very very confusion.

There are two DMIC hardware interfaces, usually referred to as PDM0 and PDM1.

There are also two dailinks in the machine driver, each configured to use different frequencies and possibly channels.

You are using DMIC0 and DMIC1 which makes it *really* hard to figure out what objects this is referring to.

I also suspect a confusion between dai and dailink.

Reply:
I guess in our case it is actually referring to the fifo A and B after the pdm's, as we have 48khz and 16khz fifos, which can take input from both pdm 0 and 1 at the same time...??? but but I think the topology dai definition is translating to a dai link as you say. Anyway I just copied the dmic0 naming, IMHO the "macro" for the name can be whatever if someone comes up with the most descriptive one.

Reply:
@plbossart are you preferring a string change here for DMIC to differentiate between the DAI link and the DMIC hardware interface ? i.e. DMIC for the DAI link and PDM for the DMIC HW ?

Reply:
Verified, works as expected except the rate, thanks @juimonen

Reply:
@plbossart I'm going to merge this for further enablement/testing and then we can rename the ambiguous DMIC DAI and HW in a subsequent topology 2 PR.  Could you give some guidance/example to @juimonen on how you see the DMIC naming and he can fix ? Thanks.